material_maker/globals.gd

```diff
@@ -30,6 +30,7 @@ const DEFAULT_CONFIG : Dictionary = {
 	auto_size_comment = true,
 	graph_line_curvature = 0.5,
 	graph_line_style = 1,
+	ui_use_native_file_dialogs = false,
```

Comment:
I suggest enabling this by default, as it's what most people expect for GUI applications these days.